test/functional/ui/statuscolumn_spec.lua

```diff
@@ -183,7 +184,7 @@ describe('statuscolumn', function()
   end)
 
   it('works with wrapped lines, signs and folds', function()
-    command("set stc=%C%s%=%{v:wrap?'':v:lnum}│\\ ")
+    command("set stc=%C%s%=%{v:virtnum?'':v:lnum}│\\ ")
```

Comment:
Since you change the command above to use long brackets, maybe change this as well:
```suggestion
    command([[set stc=%C%s%=%{v:virtnum?'':v:lnum}│\ ]])
```